ops/services/10-core/database.tf

```diff
@@ -244,6 +244,21 @@ resource "aws_cloudwatch_metric_alarm" "db_swap_usage" {
   }
 }
 
+resource "aws_cloudwatch_metric_alarm" "aurora_cpu_utilization" {
```

Comment:
Now that we're beyond merely transitioning resources from legacy to greenfield, we can be more discerning about alerts that yield to an alarm and ensure that they are actionable. What do we expect an operator or on-call to do when they encounter this alarm?